src/helpers/CheckUser.js

```diff
@@ -0,0 +1,20 @@
+import Joi from 'joi';
+import models from '../database/models';
+
+const { User } = models;
+
+export const roleEntryValidation = Joi.object({
+  RoleId: Joi.string()
+    .valid('1', '2', '3')
+    .required()
+    .pattern(/^[0-9]/)
+    .messages({
+      'string.pattern.base': '{{#label}} must contain a valid id',
+      'string.valid.base':
+        '{{#label}} must be written as 1, 2, 3 representing requester,manager, and admin respectively',
+    }),
+});
+
+export const checkUser = async (params) => {
+  return User.findOne({ where: params });
+};
```

Comment:
Please use the service already created. Although, I don't seem to find anywhere you used this function.